pkg/querier/queryrange/roundtrip.go

```diff
@@ -81,7 +81,7 @@ func newResultsCacheFromConfig(cfg base.ResultsCacheConfig, registerer prometheu
 		return nil, errors.Errorf("%s cache is not configured", cacheType)
 	}
 
-	c, err := cache.New(cfg.CacheConfig, registerer, log, cacheType)
+	c, err := cache.New(cfg.CacheConfig, registerer, log, cacheType, "cortex")
```

Comment:
any particular reason this is using "cortex" prefix?

Reply:
It's test code and the default value is still set to "cortex". If you feel strongly about it being Loki it can be changed,

Reply:
It does look nicer if they are all the same. I'll change them.

Reply:
Done